zio-http/src/main/scala/zhttp/service/client/ClientChannelInitializer.scala

```diff
@@ -1,14 +1,25 @@
 package zhttp.service.client
 
+import io.netty.channel.ChannelPipeline
 import io.netty.handler.codec.http.{HttpClientCodec => JHttpClientCodec}
+import io.netty.handler.ssl.SslContext
 import zhttp.core.{JChannel, JChannelHandler, JChannelInitializer, JHttpObjectAggregator}
 
-final case class ClientChannelInitializer[R](channelHandler: JChannelHandler) extends JChannelInitializer[JChannel]() {
+final case class ClientChannelInitializer[R](channelHandler: JChannelHandler, port: Int)
+    extends JChannelInitializer[JChannel]() {
   override def initChannel(ch: JChannel): Unit = {
-    ch.pipeline()
+    import io.netty.handler.ssl.SslContextBuilder
+    import io.netty.handler.ssl.util.InsecureTrustManagerFactory
+    val sslCtx: SslContext =
+      if (port == 443) SslContextBuilder.forClient().trustManager(InsecureTrustManagerFactory.INSTANCE).build()
```

Comment:
`InsecureTrustManagerFactory.INSTANCE ` should not be used use proper
 trustManager